python/postprocessing/modules/jme/jetmetUncertainties.py

```diff
@@ -23,8 +23,13 @@ def __init__(self, era, globalTag, jesUncertainties = [ "Total" ], jetType = "AK
         self.jesUncertainties = jesUncertainties
 
         # smear jet pT to account for measured difference in JER between data and simulation.
-        self.jerInputFileName = "Spring16_25nsV10_MC_PtResolution_" + jetType + ".txt"
-        self.jerUncertaintyInputFileName = "Spring16_25nsV10_MC_SF_" + jetType + ".txt"
+        if era == "2016":
+            self.jerInputFileName = "Summer16_25nsV1_MC_PtResolution_" + jetType + ".txt"
+            self.jerUncertaintyInputFileName = "Summer16_25nsV1_MC_SF_" + jetType + ".txt"
```

Comment:
isn't it possible to detect those from the GT names or at least to have those not hardcoded here?

Reply:
According to [L18-L21](https://github.com/cms-nanoAOD/nanoAOD-tools/blob/1995a8054ffa8738e34055668fb308d92a2f89f7/python/postprocessing/modules/jme/jetmetUncertainties.py#L18-L21) we can't derive these straight from the GT because the JES and JER global tags don't match (I've confirmed this is still true when I looked up the recommendations). So JES name comes from the global tag and the JERs are hardcoded to year.

We could hardcode a dictionary which does a JER tag lookup based on the global tag or we could build in another argument in the class for just the JER tag. Or maybe a combination of the two? Have a hard coded dictionary lookup by default but it's also possible to provide an argument for the JER tag that allows the user to force the tag without needing to change the code.